packages/engine/src/plugins/Plugin.ts

```diff
@@ -1,10 +1,11 @@
-import { ParamWithInfo } from '@store/selectors/getParamWithInfo'
-
 /**
  * Class type for a Plugin.
  */
 export interface IPlugin {
-  // new(engine: HedronEngine, useEngineStore: UseEngineStore): any;
+  /**
+   * Unique ID for plugin
+   */
+  id: string
```

Comment:
Unique ID for plugin so it can be looked up wherever needed

Reply:
I wonder if there is something good here with having a plugin override another, like someone makes a 'deluxe' lfo  plugin and still uses the id lfo. It can tap in and still work with a scene originally setup with regular lfo, and  deluxes changes at least somewhat fallback